relationships/candidates/AWSLAMBDAALIAS.yml

```diff
@@ -0,0 +1,16 @@
+category: AWSLAMBDAALIAS
+lookups:
+  - entityTypes:
+      - domain: INFRA
+        type: AWSLAMBDAFUNCTIONALIAS
+    tags:
+      matchingMode: ALL
+      predicates:
+        - tagKeys: [ "aws.Arn", "aws.arn"]
```

Comment:
tags are not case sensitive, adding only `aws.arn` should work

Reply:
Done. removed the duplicate tags